src/main/java/org/numenta/nupic/SDR.java

```diff
@@ -85,9 +85,7 @@
      * @return  the column indexes of the specified cells.
      */
     public static int[] cellsToColumns(List<Cell> cells, int cellsPerColumn) {
-        IntStream op = cells.stream().mapToInt(c -> c.getIndex());
-            
-        return op.map(cellIdx -> cellIdx / cellsPerColumn).distinct().toArray();
+        return cells.stream().mapToInt(c -> c.getColumn().getIndex()).distinct().toArray();
     }
```

Comment:
Yes that's fine. The method doesn't need the parameter so it's fine if you change the call sites too.